### ISSUE TITLE
An Introduction to Google Colaboratory

### ISSUE THREAD

**Issue by anjalibhavan** [pydatadelhi/talks#99]
- Abstract (2-3 lines)
This talk will be a hands-on tutorial about Google Colaboratory (known more popularly as just Google Colab) - an introduction, why it is needed, how to set it up, configure it for GPU usage and how to run your first program on it.

- Brief Description and Contents to be covered
1. Introduction: What is Google Colab, brief history, why it is needed, advantages
2. GPU - what is it, why needed, its availability in Google Colab
3. Tutorial: Accessing Google Colab, creating a notebook and mounting Drive, configuring GPU, running first program.

- Pre-requisites for the talk
Knowledge of Python would be a plus, although beginners are welcome.

- Time required for the talk
15-20 minutes

- Link to slides
There's no need because it's a hands-on tutorial. If it's needed I'll make some.

- Will you be doing hands-on demo as well?
Yes

- About yourself
I'm a third year undergraduate at Delhi Technological University. I have been involved in numerous applied machine learning and NLP-based research projects and endeavors over the past year, and am also a member of PyLadies Delhi. 

- Are you comfortable if the talk is recorded and uploaded to PyData Delhi's YouTube channel ?
Yes

**Comment by shagunsodhani** [comment]
Hey @anjalibhavan thank you for the proposal. Looking forward to the talk.

**Comment by anjalibhavan** [comment]
Thank you! Would this be for the Pydata Delhi conference in August?

**Comment by shagunsodhani** [comment]
I think @MSanKeys963 should answer that.

**Comment by sanketverma1704** [comment]
No @anjalibhavan. These talks issues are for the meetup. For the conference, we will open the CFP separately.

**Comment by anjalibhavan** [comment]
Oh alright! The next meetup is on 11th, if I'm correct? My availability on that day is not yet certain, but I would like to give this talk in maybe the next after this in case I'm unable to make it on 11th.

**Comment by sanketverma1704** [comment]
Yes, the next meetup is on 11th. But I think we already have outstanding speakers for that. You can RSVP [here](https://www.meetup.com/PyDataDelhi/events/261057427/).

**Comment by anjalibhavan** [comment]
Oh alright. I'll give a talk in a later meetup then. :)

**Comment by sanketverma1704** [comment]
Hi, @anjalibhavan. Can you update the slides, please?

**Comment by sanketverma1704** [comment]
Hi @anjalibhavan. Would you like to deliver this in our upcoming meetup?

**Comment by sanketverma1704** [comment]
Soft reminder, @anjalibhavan.

**Comment by anjalibhavan** [comment]
Hi. Apologies for not responding on time. I won't be free to attend the upcoming meetup, sorry. I will post another talk issue as well later, hopefully to speak at some other meetup in the future.
